crates/forge/tests/it/test_helpers.rs

```diff
@@ -90,10 +89,10 @@ impl ForgeTestProfile {
     pub fn zk_project(&self) -> ZkProject {
         let zk_config = self.zk_config();
         let mut zk_project =
-            foundry_zksync_compiler::create_project(&zk_config, zk_config.cache, false)
+            foundry_zksync_compiler::config_create_project(&zk_config, zk_config.cache, false)
                 .expect("failed creating zksync project");
-        zk_project.paths.zksync_artifacts = zk_config.root.as_ref().join("zk").join("zkout");
-        zk_project.paths.zksync_cache = zk_config
+        zk_project.paths.artifacts = zk_config.root.as_ref().join("zk").join("zkout");
```

Comment:
Does this finally mean that `forge clean` will clean zkout?

Reply:
No, but it's not that hard to add support if we wanted to as calling `project.cleanup()` on the zk_project should work as intended, deleting all zk stuff

Reply:
Alright - I think we might want to do so, just not in this PR